Marlin/src/MarlinCore.cpp

```diff
@@ -1013,11 +1013,31 @@ void setup() {
   SETUP_RUN(HAL_init());
 
   // Init and disable SPI thermocouples
-  #if HEATER_0_USES_MAX6675
-    OUT_WRITE(MAX6675_SS_PIN, HIGH);  // Disable
+  #if HEATER_0_USES_MAX6675 && PIN_EXISTS(MAX6675_SS)
```

Comment:
Is `HEATER_0_USES_MAX6675` without a `MAX6675_SS_PIN` a valid combination? If not, we should put a sanity check for that, instead of add `&& PIN_EXISTS(MAX6675_SS_PIN)` here.

Reply:
HEATER_0_USES_MAX6675 is only set when TEMP_SENSOR_0 == -5 || TEMP_SENSOR_0 == -3 || TEMP_SENSOR_0 == -2

So without the "&& PIN_EXISTS(MAX6675_SS)"  the code would try to disable the MAX6675_SS_PIN when that pin was not defined because MAX31865_CS_PIN was used

Reply:
Put compatibility layer in place so this is no only using CS_PIN for the MAX boards